src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1101,6 +1102,8 @@ def update_connected_cluster(cmd, client, resource_group_name, cluster_name, htt
         cmd_helm_upgrade.extend(["--set", "global.isCustomCert={}".format(True)])
     if container_log_path is not None:
         cmd_helm_upgrade.extend(["--set", "systemDefaultValues.fluent-bit.containerLogPath={}".format(container_log_path)])
+    if enable_oidc_issuer:
```

Comment:
how are we supporting disabling of this feature?

Reply:
We do not currently have plans to support disabling of this feature as far as I am aware.